src/Native/Runtime/PalRedhawkCommon.h

```diff
@@ -167,6 +167,13 @@ typedef void (__fastcall * ThreadExitCallback)();
 extern ThreadExitCallback g_threadExitCallback;
 
 typedef Int32 (*PHARDWARE_EXCEPTION_HANDLER)(UIntNative faultCode, UIntNative faultAddress, PAL_LIMITED_CONTEXT* palContext, UIntNative* arg0Reg, UIntNative* arg1Reg);
+
+#else
+
+typedef void (__stdcall * ThreadExitCallback)();
```

Comment:
Move this outside the ifdef instead

Reply:
Should we care about calling convention for this callback? I've realized after I pushed changes that there is now `__fastcall` for both Unix and Windows.

Reply:
It is best to just use the default platform calling convention everywhere. You can delete the explicit `stdcall` or `fastcall` on this.

Reply:
Ah I already found explicit calling convention omitted in runtimelab file.